docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
+services:
+ sourcecred:
+  container_name: sourcecred
+  image: sourcecred:latest
+  build:
+   context: .
+   dockerfile: docker/Dockerfile
+   cache_from:
```

Comment:
For local builds this is redundant. And for CI builds you'll still need to manually pull the images, so declaring the cache_from there isn't much extra work.